.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,95 @@
+name: Pre-release
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  pre-release:
+    runs-on: ubuntu-latest
+    name: Build vsix and upload to GitHub pre-release
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: Install .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: "8.0"
+
+      - name: Get current version
+        run: |
+          VERSION=$(node -p "require('./package.json').version")
+          echo VERSION=$VERSION >> $GITHUB_ENV
+          echo VERSION_TAG=v$VERSION >> $GITHUB_ENV
```

Comment:
@domi-b Nach Rücksprache mit @Philippluca führen wir die Revision in der Version ein, so sparen wir uns das Aktualisieren eines womöglich vorhandenen Pre-Releases und erstellen bei jedem Change auf dem Hauptentwicklungszweig eine neue Pre-Release-Version (wie in anderen Projekten üblich). Können es gerne gemeinsam anschauen, meld' dich einfach bei mir.